### PR TITLE
added gzip.md

### LINKED ISSUE #101: gzip

### PR DESCRIPTION
**Pull request by tayzheng**
# Advanced HW 9 submission

Closes #101 gzip

Name: Taylor Zheng
Uniqname: tayzheng

### CODE REVIEW

File: _commands/basics/gzip.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+---
+
+gzip
+-------
+
+Gzip reduces the size of the name files by compressing ( or "packaging") a file into an "archive file". It will only attempt to compress regular files. Compressed files can be restored to their original form using gzip -d or gunzip or zcat.
```
Comment (ppannuto) [inline _commands/basics/gzip.md]: Careful not to confuse people here, `tar` is an archive-creating utility, `gzip` is a compression utility. Technically gzip _can_ create archives, but it's almost never done (unlike `zip`, which is both a compression and archive format). I wouldn't use the word archive here.

**Comment by ppannuto** [comment]
It would be good to add to the examples some indication of what `gzip` is actually doing, perhaps showing the output of `ls -lh` to show the reduction in file size. Relatedly, explaining gzip's levels (i.e. `gzip -1` vs `gzip -9`) and the tradeoffs they present could be useful.

`-k` is a really useful example to show as well since it can be a little surprising when gzip deletes your source file on you by default.

**Comment by the-alex** [comment]
Unfortunately, the submission doesn't render well for me.

screen shot 2016-03-25 at 5 55 24 pm

**Comment by themoh** [comment]
We'll be closing all pull requests at midnight tonight, so please have this done by then.

**Comment by the-alex** [comment]
There are still rendering issues with this submission. If you test locally, you will see them. Here's what this submission currently renders like on my machine with the most recent commit ...
screen shot 2016-03-27 at 6 45 55 pm

**Comment by the-alex** [comment]
The reason these are rendering weirdly is because you're using the `####` lines to separate the examples, when you should be using them as headers. 

Ex.

------
Good ...
#### RAWR

Bad ...
####
rawr


-----

raw markdown for Ex. above ...
~~~markdown
Good ...
#### RAWR

Bad ...
####
rawr
~~~

File: _commands/basics/gzip.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+---
+gzip
+-------
+
+'gzip' reduces the size of the name files by compressing a file. It will only attempt to compress regular files. Compressed files can be restored to their original form using gzip -d or gunzip or zcat.
```
Comment (themoh) [inline _commands/basics/gzip.md]: change to `gzip`

Also, use backticks " ` " around all mentioning of gzip.

**Comment by themoh** [comment]
Please add in bullet points as well (see "sed.md" for an example on how to format these)

**Pull request merged**